contrib/ci/Jenkinsfile

```diff
@@ -1,27 +1,48 @@
+def lualint() {
+	sh 'luacheck package scripts targets'
+}
+
+def shelllint() {
+	sh 'find -not \( -path './*.git/*' -prune \) -not \( -path './package/*' -prune \) -not \( -path './scripts/*' -prune \) -type f -exec bash -c 'if echo $1 | grep -qE ".*\.sh$" || head -n1 $1 | grep -qE "^#.*(sh|bash|dash|ksh)$"; then shellcheck -f gcc $1; fi' _ {} \;'
+	sh 'find package -type f -exec bash -c 'if echo $1 | grep -qE ".*\.sh$" || head -n1 $1 | grep -qE "^#.*(sh|bash|dash|ksh)$"; then shellcheck -x -f gcc -s sh -eSC2039,SC1091,SC2155,SC2034 $1; fi' _ {} \;'
+	sh 'find scripts -type f -exec bash -c 'if echo $1 | grep -qE ".*\.sh$" || head -n1 $1 | grep -qE "^#.*(sh|bash|dash|ksh)$"; then shellcheck -f gcc -x -e SC2154,SC1090,SC2181,SC2155,SC2148,SC2034,SC2148  $1; fi' _ {} \;'
```

Comment:
Rather than defining these commands in the Jenkinsfile, we should add them to Gluon's main Makefile, so developers can easily run the same checks.

I propose adding make targets `lint-lua`, `lint-sh`, and a `lint` target that runs both.

Reply:
Shall I do it in this MR or in a separated one?

Reply:
Just do everything in this PR.

Reply:
I think as make comands ist not that easy, because there seems some predefines necessary like:
`targets.mk:2: *** GLUON_DEPRECATED not set. Please consult the documentation.  Stop.`
same like `GLUON_RELEASE` there muss be a valid side dir as well.

Reply:
ok, I have an idea. I will create sh files like `contrib/ci/lint-sh.sh` so they can be called by both via make and jerkins and thous files can be also linted.

Reply:
I have pushed my changes

Reply:
- Don't put the scripts into `contrib/ci`, this is not only about CI. `scripts` would be more appropriate.
- I'd still prefer to have make targets as described above (making the scripts an implementation detail - CI would also use make). I don't see a problem with making a checked out up-to-date site repo a prerequisite, it is already available in CI for build tests.

Reply:
I,ve moved to scripts but I don't understand what do you mean with "... have make targets ... ". Did you would like to have a list of possible linters ? something like `make lint` to show a list of possible linters and `make lint all` to run all but also have `make lint sh` to run specific ones?

Reply:
No, just the make targets `lint`, `lint-lua` and `lint-sh` as described before.

Reply:
So it should be called `make targets lint` or `make targets lint-sh` that don´t make sense to me thus the linters are not target specific I prefer `make lint` or `make lint-sh` like it is at the moment.

Reply:
`make lint` and `make lint-sh` is exactly what I want. The arguments `lint` and `lint-sh` are Makefile targets.

Reply:
ok, and what should I change now? I should be now like you want. So it is resolve now?